test/testdata/ingress/reaper-ingress.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: traefik.containo.us/v1alpha1
 kind: IngressRoute
```

Comment:
I think it would be better to use native Ingress when we can. Not a concern for this PR but I wanted to mention it since I was looking at this.

Reply:
I agree in principle, but the issue is how to route raw TCP traffic in order to test Stargate's CQL api. Can native ingress handle that? I was under the assumption that it currently can't.

Reply:
>I was under the assumption that it currently can't.

I think this is correct. How we handle Ingress is definitely a discussion that needs to happen outside of this PR. The fact that we have _something_ which allows us to test APIs is 👍